python/requirements.txt

```diff
@@ -1,5 +1,5 @@
 crash
 crate
-flask==0.10
+flask==0.12.3
```

Comment:
would it make sense to bump flask to it's latest `1.0.2` release? is there a lot of rewrite necessary?